job.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: helloworld-apps-job
+  labels:
+    strimzi.io/kind: topic
+    strimzi.io/cluster: my-cluster
+data:
+  name: helloworld-apps-job
+  partitions: "3"
+  replicas: "3"
+  config: |-
+    {
+      "retention.bytes": "1073741824",
+      "retention.ms": "86400000",
+      "segment.bytes": "1073741824"
+    }
+---
+apiVersion: batch/v1
+kind: Job
```

Comment:
Lets make our life easier and get rid of the job.yaml ... I used it only for OpenShift Online because I didn't had any free resources for deployment :-).